etc/base-secrets.yaml

```diff
@@ -175,3 +175,13 @@ velero:
           [default]
           aws_access_key_id=change_me
           aws_secret_access_key=change_me
+
+nifi:
```

Comment:
perhaps these secrets can be initialised in the `bin/generate-secrets` script (just add lines for specific values using `insert_secret`)

Reply:
The script will auto generate any secret that has the value of `secret`

Reply:
ok great, thanks